crates/block-producer/src/lib.rs

```diff
@@ -25,10 +25,10 @@ pub use server::BlockProducer;
 pub const COMPONENT: &str = "miden-block-producer";
 
 /// The number of transactions per batch.
-const SERVER_MAX_TXS_PER_BATCH: usize = 2;
+pub const SERVER_MAX_TXS_PER_BATCH: usize = 8;
 
 /// Maximum number of batches per block.
-const SERVER_MAX_BATCHES_PER_BLOCK: usize = 4;
+pub const SERVER_MAX_BATCHES_PER_BLOCK: usize = 8;
```

Comment:
These are now defaults (i.e., these can be overridden) - right? If so, would `DEFAULT_MAX_TXS_PER_BATCH` and `DEFAULT_MAX_BATCHES_PER_BLOCK` be better names?

Reply:
We could also remove them entirely? The default values are really more of a node cli thing imo.

Reply:
Yep, that makes sense. Would they just move to the `miden-node` crate?

Reply:
Yeah, in fact we already have a couple there in `commands/mod.rs`.

I'll update the follow-up issue to make this more of a standard.